students/ghassan/lesson04/new_mailroom.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import sys
+
+global donors
+donors = {
```

Comment:
No need to declare `global` here, since `donors` is defined in the global namespace it becomes a global variable by default